src/docs/get-started/flutter-for/ios-devs.md

```diff
@@ -1196,9 +1196,9 @@ respective build systems.
 While there is a Podfile in the iOS folder in your
 Flutter project, only use this if you are adding native
 dependencies needed for per-platform integration. In general, use
-`pubspec.yaml` to declare external dependencies in Flutter. A good place to
-find great packages for Flutter is
-[Pub](https://pub.dartlang.org/flutter/packages/).
+`pubspec.yaml` to declare external dependencies in Flutter.
+A good place to find great packages for Flutter is
+[Pub]({{site.pub}}/flutter/packages/).
```

Comment:
[Pub] -> the [Pub site]
? I think we want to differentiate between the pub tool and the Pub site.